components/tikv_kv/src/lib.rs

```diff
@@ -93,6 +94,100 @@ impl Modify {
     }
 }
 
+impl From<Modify> for raft_cmdpb::Request {
```

Comment:
Why move it here?

Reply:
I'm sorry that I am lazy here. I will revert them.

_`Modify` to `Request` is used by the codes in this PR before. But I do not revert them because they will be used in the coming PR. `components/tikv_kv/src/rocksdb_engine.rs` will use it to call "pre_propose_query" observers for uni tests [here](https://github.com/tikv/tikv/pull/12128/files#diff-10c2f3682d5777d8dca294378acff71faa16589fabba789532041ee3db10cad7R211)._